pkg/config/serving/options.go

```diff
@@ -0,0 +1,47 @@
+package serving
+
+import (
+	"net"
+	"strconv"
+
+	genericapiserveroptions "k8s.io/apiserver/pkg/server/options"
+	utilflag "k8s.io/apiserver/pkg/util/flag"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+func ToServingOptions(servingInfo configv1.HTTPServingInfo) (*genericapiserveroptions.SecureServingOptionsWithLoopback, error) {
+	host, portString, err := net.SplitHostPort(servingInfo.BindAddress)
```

Comment:
do we validate that BindAddress is actually an IP? Otherwise, we get a nil IP in line 24 I believe, not what is intended.

Reply:
> do we validate that BindAddress is actually an IP? Otherwise, we get a nil IP in line 24 I believe, not what is intended.

it will eventuallly.  it does not today.